pkg/telemetry/telemetry.go

```diff
@@ -2,40 +2,14 @@ package telemetry
 
 import (
 	"bytes"
-	"encoding/json"
 	"net/http"
-	"runtime"
 
-	"github.com/cloudskiff/driftctl/pkg/analyser"
-	"github.com/cloudskiff/driftctl/pkg/version"
+	"github.com/cloudskiff/driftctl/pkg/memstore"
 	"github.com/sirupsen/logrus"
 )
 
-type telemetry struct {
-	Version        string `json:"version"`
-	Os             string `json:"os"`
-	Arch           string `json:"arch"`
-	TotalResources int    `json:"total_resources"`
-	TotalManaged   int    `json:"total_managed"`
-	Duration       uint   `json:"duration"`
-}
-
-func SendTelemetry(analysis *analyser.Analysis) {
-
-	if analysis == nil {
-		return
-	}
-
-	t := telemetry{
-		Version:        version.Current(),
-		Os:             runtime.GOOS,
-		Arch:           runtime.GOARCH,
-		TotalResources: analysis.Summary().TotalResources,
-		TotalManaged:   analysis.Summary().TotalManaged,
-		Duration:       uint(analysis.Duration.Seconds() + 0.5),
-	}
-
-	body, err := json.Marshal(t)
+func SendTelemetry(s memstore.Store) {
+	body, err := s.Bucket(memstore.TelemetryBucket).MarshallJSON()
```

Comment:
We should not call `MarshallJSON` directly, use `json.Masrshall(bucket)` instead.

Btw, I'm wondering if we should pass directly bucket content, I think this is not the responsibility of the bucket to be serialized directly. I think we should still build a `Telemetry` object with explicit types and then serialize it to control serialization type. I see a weakness here if by mistake you put the wrong type in a `bucket.Set`